lib/components/colors.dart

```diff
@@ -30,13 +33,96 @@ class Colors implements AnsiColorType {
   static const highCyan = Colors._(96, 106);
   static const highWhite = Colors._(97, 107);
 
+  static const mapping = <int, Colors>{
+    30: black,
+    31: red,
+    32: green,
+    33: yellow,
+    34: blue,
+    35: magenta,
+    36: cyan,
+    37: white,
+    90: highBlack,
+    91: highRed,
+    92: highGreen,
+    93: highYellow,
+    94: highBlue,
+    95: highMagenta,
+    96: highCyan,
+    97: highWhite,
+    40: black,
+    41: red,
+    42: green,
+    43: yellow,
+    44: blue,
+    45: magenta,
+    46: cyan,
+    47: white,
+    100: highBlack,
+    101: highRed,
+    102: highGreen,
+    103: highYellow,
+    104: highBlue,
+    105: highMagenta,
+    106: highCyan,
+    107: highWhite,
+  };
+
+  static const stringMapping = <int, String>{
+    30: 'black',
```

Comment:
You could consider adding the string to the object itself as `final String name` or `ansiName`.

Then the usages become `name` instead of `_colorFromCode(bgCode)` in `printableFg` etc.